easybuild/tools/build_log.py

```diff
@@ -160,6 +180,15 @@ def init_logging(logfile, logtostdout=False, testing=False):
     return log, logfile
 
 
+def get_eb_logger(name=None, *args, **kwargs):
```

Comment:
if you are going to trust the `get_eb_logger`, you might as well not need trust it and return the EasyBuildLog equivalent of getLogger (and not modify EasyBuildLog)